.github/workflows/cli-docker-build.yml

```diff
@@ -0,0 +1,63 @@
+name: Kinto CLI Build and Release Binaries
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        go_version: [1.13.x]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go_version }}
+
+      - name: Set CLI_VERSION
+        run: echo "CLI_VERSION=$(echo ${GITHUB_REF} | rev | cut -d'/' -f 1 | rev )" >> $GITHUB_ENV
+
+      - name: Configure & Build
+        uses: crazy-max/ghaction-xgo@v1
+        with:
+          xgo_version: latest
+          go_version: ${{ matrix.go_version }}
+          dest: build
+          prefix: kinto
+          targets: windows/amd64,linux/amd64,darwin/amd64
+          v: true
+          x: false
+          ldflags: -X "github.com/kintoproj/kinto-cli/internal/config.Version=${{ env.CLI_VERSION }}"
```

Comment:
I;m not sure if the flags will work this way. the dir structure needs to be changed since its a monorepo now

Reply:
okay. so I'll change to: `github.com/kintoproj/kintohub/cli/internal/config.Version=${{ env.CLI_VERSION }}`

WDYT?

Reply:
should work. will need to test though

Reply:
done